.github/command-docs.md

```diff
@@ -58,6 +58,7 @@
 | .abbreviations  | Links to most commonly used abbreviations                                                                                                                                                                                                                                   | .abb <br> .abrv                                    |
 | .airac          | Provides information about free SimBrief account AIRAC limitations                                                                                                                                                                                                          | ---                                                |
 | .build          | Link to docs for build info                                                                                                                                                                                                                                                 | .build info <br> .buildinfo                        |
+| .brakecheck     | Provides instructions to find the current brake pedal inputs                                                                                                                                                                                                                | .brake
```

Comment:
This needs to go above `.build` to adhere to the alphabetical ordering.

You're missing the `|` at the end of the table